Dockerfile

```diff
@@ -16,7 +16,7 @@ WORKDIR /opt/killrvideo-generator
 # Copy package.json for dependencies
 COPY package.json /opt/killrvideo-generator/
 COPY npm-shrinkwrap.json /opt/killrvideo-generator/
-    
+
```

Comment:
Did this file actually change (beyond whitespace?)

Reply:
Seems strange. I believe it had an ADD before vs. the COPY but it sure doesn't show it.  This worked last I tired it, so it seems ok regardless.